content/en/real_user_monitoring/guide/monitor-hybrid-react-native-applications.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Monitor Hybrid React Native Applications
+kind: guide
+description: Guide for monitoring hybrid React Native applications.
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM Monitors'
+---
+
+## Overview
+
+React Native is a JavaScript framework for developing hybrid mobile applications that can run natively on both Android and iOS.
+
+If you have a hybrid application that is built on React Native, you can use Datadog to monitor the same application both from the React Native and native sides.
```

Comment:
```suggestion
If you have a hybrid application that is built on React Native, you can use Datadog to monitor the same application from both the native Android or iOS side and the React Native side.
```

What do you think about this? I don't think it needs clarification every time, but maybe once up front?

Reply:
agreed, thanks